static/css/style.css

```diff
@@ -1,166 +1,170 @@
-:root {

```

Comment:
what's happended here? Did you replace the whole file?

Reply:
Nope, not sure why it's showing like that there. I added a class but didn't change anything else in the file. I had to re-add the class once I pulled and merged the latest PR, so maybe that may have thrown git? I've checked the repo though and all the existing classes are still there, as well as my new class.